packages/webapp/src/containers/AddCropVariety/AddCropVariety.jsx

```diff
@@ -31,8 +31,8 @@ function AddCropVarietyForm({ history, match }) {
       ...persistedFormData,
       ...data,
       crop_variety_name: data.crop_variety_name.trim(),
-      crop_varietal: data.crop_varietal.trim(),
-      crop_cultivar: data.crop_cultivar.trim(),
+      crop_varietal: data.crop_varietal.length != 0 ? data.crop_varietal.trim() : null,
+      crop_cultivar: data.crop_cultivar.length != 0 ? data.crop_cultivar.trim() : null,
```

Comment:
This is also something I had missed.
When `interested` (line 18) is true, `onSubmit` will not be called and `onContinue` (line 25) will be called instead, and these string will not be trimmed. (The function that will be called is onSubmit in `/packages/webapp/src/containers/AddCropVariety/ComplianceInfo.jsx`) Do we want to handle it?

A little more input:
- `data.crop_varietal.length ? data.crop_varietal.trim() : null` seems more natural to me

Reply:
Just did some testing, it seems atleast within the scope of this ticket the trim is not actually required